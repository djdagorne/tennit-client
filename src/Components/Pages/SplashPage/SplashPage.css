.popup {
  position: fixed;
  top: 0;
  left: 0;
  right: 0;
  bottom: 0;
  background-color: rgba(0,0,0, 0.5);
  z-index: 10;
}

.popup_inner {
  overflow-y: scroll;
  height: 300px;
  width: 260px;
  margin: auto;
  padding-left: 10px; /* offset the scrollbar width, to keep form centered */
  margin-top: 25vh;
  border-radius: 10px;
  background-color: #F2C49B;
  z-index: 11;
}

.popup_inner h2{
  margin: 0;
  margin-top: 20px;
}

.slogan{
  filter:blur(0px);
  display: block;
  margin-top: 33vh;
  margin-bottom:50vh; /* messing with this moves the footer :S idk */
  padding: 0;
  color: #F0F0F2;
  text-shadow: 0px 3px 10px rgba(119, 119, 119, 0.493);
}

h1.slogan-text{
  font-family: 'Raleway', sans-serif;
  margin: 0;
  padding: 0;
  font-size: 6em;
  font-weight: 700;
}

h2.slogan-text{
  font-family: 'Raleway', sans-serif;
  margin: 0;
  padding: 0;
  font-size: 2.2em;
  font-weight: 100;
  line-height: 0;
  letter-spacing: 4px; 
}

.sign-up-button{
    margin-top: 20px;
    border-radius: 5px;
    font-size:20px;
  }

.splash-background{  
  position: fixed;
  top: 0;
  left: 0;
  right: 0;
  bottom: 0;
  filter:blur(2px);
  background-image: 
    linear-gradient(
      rgba(136, 136, 136, 0.3), /* tint the color to match theme  */
      rgba(134, 134, 134, 0.158)
    ),
    url("../../../imgs/IMG_1745.jpg");
  background-position: center;
  background-size: cover;
  z-index: -1;
}