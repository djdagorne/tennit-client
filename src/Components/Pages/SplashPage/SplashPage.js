import React, { Component} from 'react'
import './SplashPage.css'
import TennitContext from '../../../TennitContext'

export default class SplashPage extends Component {
    static contextType = TennitContext

    render(){
        return (
            <div className="splash-wrap">
                <div className="title-box">
                    <div className="slogan">
                        <h1 className="slogan-text">Fall in love.</h1> 
                        <h2 className="slogan-text">with affordable rent.</h2>
                    </div>
                </div>
                <div className="intro-box">
                    <h3 className="intro-text">
                            <p className="opening-line">
                                Finding a soulmate can take a lifetime.
                            </p>
                            <p >
                                But you need to find someone who will split your bachelor apartment by the first.
                            </p>
                            <p >
                                ...and they got to be okay with your couch.
                            </p>
                            <p >
                                ...and they should probably know about your cat, Sprinkles.
                            </p>
                            <p >
                                Starting today, you can find love that works for you, your couch AND Sprinkles. 
                            </p> 
                            <p>
                                And you set the price.
                            </p>
                        <button className="splash-button" type="submit" onClick={e=>this.context.togglePopup('create')}>Make an Account Today</button>

                    </h3>
                </div>
            </div>
        );
    }
}