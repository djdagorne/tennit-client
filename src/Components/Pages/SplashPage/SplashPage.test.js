import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import SplashPage from './SplashPage';
import '../../../setupTest';



it('renders without crashing', () => {
    const wrapper = shallow(<SplashPage />);
    expect(toJson(wrapper)).toMatchSnapshot();
})
