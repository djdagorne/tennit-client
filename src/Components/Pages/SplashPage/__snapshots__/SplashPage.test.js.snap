// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="splash-wrap"
>
  <div
    className="title-box"
  >
    <div
      className="slogan"
    >
      <h1
        className="slogan-text"
      >
        Fall in love.
      </h1>
      <h2
        className="slogan-text"
      >
        with affordable rent.
      </h2>
    </div>
  </div>
  <div
    className="intro-box"
  >
    <h3
      className="intro-text"
    >
      <p
        className="opening-line"
      >
        Finding a soulmate can take a lifetime.
      </p>
      <p>
        But you need to find someone who will split your bachelor apartment by the first.
      </p>
      <p>
        ...and they got to be okay with your couch.
      </p>
      <p>
        ...and they should probably know about your cat, Sprinkles.
      </p>
      <p>
        Starting today, you can find love that works for you, your couch AND Sprinkles.
      </p>
      <p>
        And you set the price.
      </p>
      <button
        className="splash-button"
        onClick={[Function]}
        type="submit"
      >
        Make an Account Today
      </button>
    </h3>
  </div>
</div>
`;
